cloud-manager/README.md

```diff
@@ -0,0 +1,58 @@
+# Cloud Manager CLI tool
+Initially, this tool is built for rotating the kubernetes nodes to deploy a new AMI.
+
+## Build
+To make the binary run:
+`make build`
+## Usage
```

Comment:
nit: seems like we want an empty line between these two lines

Similar comment for other lines below.